reg_form_ext/tests.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+from django.conf import settings
+from django.test import TestCase
+from django.test.client import Client
+from django.test.utils import override_settings
+from django.core.urlresolvers import reverse
+
+from .forms import PhoneInfoForm
+
+
+@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
+@override_settings(REGISTRATION_EXTENSION_FORM='reg_form_ext.forms.PhoneInfoForm')
```

Comment:
Interesting.. you didn't have to set `ADDL_INSTALLED_APPS` here as the [upstream code suggested](https://github.com/edx/edx-platform/pull/10413/files#diff-82bda515879c96ab0641bfcbe58631c3R2672). 

I also manually tested that no `ADDL_INSTALLED_APPS` change was required to run this manually in my ginkgo devstack, so I think it's safe to remove that from your PR description as well.

Reply:
And thank you for adding tests 😄

Reply:
Were you able to run the migrations without including the `ADDL_INSTALLED_APPS` line? I wasn't able to run the migrations without it. So I could load the form, but not submit it.

Ah, this works in the tests because I realize now my tests never saved the form, and so don't test the database or models. I've tried adding a couple tests to rectify that.
However, I can't seem to modify INSTALLED_APPS before the test database is created. So the migrations from this module are not included in the test database.

I tried a few different methods:
* Running syncdb from within `setUpClass()`, based on [this snippet](https://djangosnippets.org/snippets/2843/)
* Adding a `reg_form_ext/test_settings.py`, and configuring that as the `DJANGO_SETTINGS_MODULE` in pytest.ini
* Creating `conftest.py` and overriding pytest fixtures from there including `django_db_setup`, and `pytest_configure`. It seems from [this comment](https://github.com/pytest-dev/pytest-django/issues/33#issuecomment-18058652) that this strategy may not be possible.

It works just fine if I add `INSTALLED_APPS.append('reg_form_ext')` to `lms/envs/test.py`, but that doesn't seem like a reasonable part of the test workflow.

Reply:
@kaizoku Ach you're right -- `ADDL_INSTALLED_APPS` has to be set to run the migrations. 

Re your tests, I tried a bunch of things too, and nothing worked 😦 

Could you modify your tests to make them mock out the storage part, like the upstream PR did? https://github.com/edx/edx-platform/pull/10413/files#diff-9c15f7dd12df13f452e1c32f8b68a166R1470